packages/contracts/deploy/014-OVM_L1StandardBridge.deploy.ts

```diff
@@ -86,7 +86,7 @@ const deployFn: DeployFunction = async (hre) => {
 
   // Check if if we're on the hardhat chain ID. This will only happen in CI. If this is the case, we
   // can skip directly to transferring ownership over to the ChugSplashDictator contract.
-  if (isHardhatNode(hre)) {
+  if (await isHardhatNode(hre)) {
```

Comment:
Were these all wrong before and using promises instead?

Reply:
This specific step was wrong, yes. We caught it while manually reviewing this.